fastlane/lib/fastlane/actions/current_build_number.rb

```diff
@@ -0,0 +1,141 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      LATEST_BUILD_NUMBER = :LATEST_BUILD_NUMBER
+    end
+
+    class CurrentBuildNumberAction < Action
+      def self.run(params)
+        require 'spaceship'
+        credentials = CredentialsManager::AccountManager.new(user: params[:username])
```

Comment:
This is not necessary, check out the other actions like `DownloadDsymsAction`. I just checked, and this code has been here before, but let's get rid of it and do like `DownloadDsymsAction`